pkg/apis/monitoring/v1/types.go

```diff
@@ -1417,7 +1419,7 @@ type ProberSpec struct {
 	// Defaults to `http`.
 	Scheme string `json:"scheme,omitempty"`
 	// Path to collect metrics from.
-	// Defaults to `/probe`.
+	// If empty, Prometheus uses the default value (e.g. `/probe`).
```

Comment:
Ah sorry for the incorrect reply. For this one I think we can rely on a default value in the OpenAPI spec since the operator already sets the value [explicitly](https://github.com/prometheus-operator/prometheus-operator/blob/17618aa437b6d284c7d5e310e1ab0eb67d6427c6/pkg/prometheus/promcfg.go#L951-L954).